mlflow/tracking/_tracking_service/client.py

```diff
@@ -230,6 +231,11 @@ def log_batch(self, run_id, metrics=(), params=(), tags=()):
             _validate_tag_name(tag.key)
         self.store.log_batch(run_id=run_id, metrics=metrics, params=params, tags=tags)
 
+    def record_logged_model(self, run_id, mlflow_model):
+        if not isinstance(mlflow_model, Model):
+            raise Exception("wrong model")
```

Comment:
Could we have a more informative message here?

Reply:
Oh yeah, I meant to fix this one, thx for the catch